Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -153,16 +197,20 @@ class SearchSettingsTableViewController: ThemedTableViewController {
     }
 
     override func numberOfSections(in tableView: UITableView) -> Int {
+        if (LegacyFeatureFlagsManager.shared.isFeatureEnabled(.firefoxSuggestFeature, checking: .buildAndUser)){
```

Comment:
```suggestion
        if (LegacyFeatureFlagsManager.shared.isFeatureEnabled(.firefoxSuggestFeature, checking: .buildAndUser)) {
```
swiftlint will fail here, otherwise